docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -519,6 +521,32 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
 
     noIndentation(actualYaml) shouldBe load("expected_extensions.yml")
   }
+
+  test("should match the expected json with schema dialect") {
```

Comment:
as these are not tests of the openapi interpreter, but for the serialisation of customly enhanced `OpenAPI` objects, I'd move those to the `openapi-model` module

Reply:
or rather, since we are testing serialisation, the appropriate place would be `openapi-circe-yaml` - where the encoders/decoders for the `OpenAPI` data type are defined

Reply:
Moved to module `openapi-circe` - the encoders/decoders are there :)